tests/Events/EventsDispatcherTest.php

```diff
@@ -571,13 +571,6 @@ public function testInvokeIsCalled()
         $d->dispatch('myEvent', 'somePayload');
         $this->assertEquals(['__construct', '__invoke_somePayload'], $_SERVER['__event.test']);
 
-        // It throws an "Error" when there is no method to be called.
-        $d = new Dispatcher;
-        $d->listen('myEvent', TestListenerLean::class);
-        $this->expectError();
-        $this->expectErrorMessage('Call to undefined method '.TestListenerLean::class.'::__invoke()');
-        $d->dispatch('myEvent', 'somePayload');
-
```

Comment:
Unfortunately we needed to remove this test because there's no way to assert Error exceptions anymore in PHPUnit. See https://github.com/sebastianbergmann/phpunit/issues/5062

Reply:
No because it would still throw the warning: https://github.com/sebastianbergmann/phpunit/blob/9.6/src/Framework/TestCase.php#L607

Expecting Error, Warning, Notice or Deprecation is simply not possible anymore.

Reply:
It's a bit hacky maybe, but you could override the exception handler and do-it-yourself.

Reply:
I don't want to go there. Think it's best to follow PHPUnit's defaults as much as possible.

Reply:
Nvm, seems I was looking at a wrong error type.